CDCgov/NEDSS-Modernization
# CNFT2-1481 classic preview

## Description

In the modernized page builder, a link needs to exist to allow the user to view a page in the Classic display. This PR adds a `Classic preview` button as well as handles the routing to load the Classic preview page.

If the user clicks the `Edit Page` button on the Classic page, the user will be redirected back to the Modernized edit page. Other buttons are not yet routed from Classic.

## Tickets

* [CNFT2-1481](https://cdc-nbs.atlassian.net/browse/CNFT2-1481)

## Steps to verify

1. Restart nbs-gateway, question-bank applications
2. Navigate to [NBS](http://localhost:8080/nbs/login)
3. Log in as `superuser`
4. Click System Management -> Page Management -> Manage Pages
5. Click and Investigation page from the table
6. Click the new `Eye`con 
7. Verify a new tab is opened and the correct page is loaded in the Classic preview
8. Click the `Edit Page` button
9. Verify the Modernized edit page is loaded

![Nov-22-2023 17-51-50](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/6880a91f-a73d-4f6f-9ae6-9c76fdca89b6)


[CNFT2-1481]: https://cdc-nbs.atlassian.net/browse/CNFT2-1481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
